transform/mattermost-analytics/models/intermediate/product/licenses/_licenses_servers/int_cloud_servers.sql

```diff
@@ -0,0 +1,9 @@
+select timestamp::date as installation_date
+        , installation_id as installation_id
+        , server_id as server_id
+        , license_name as license_name
+    from {{ ref('stg_mm_telemetry_prod__license')}} 
+    where installation_id is not null
```

Comment:
@catalintomai I am using this filter for cloud across the code in this PR, to be consistent with what Nick shared.